### ISSUE TITLE
Umbraco 13 support, with blocks in RTE

### ISSUE THREAD

**Issue by jeroenmink2** [abjerner/Limbo.Umbraco.Rte#1]
After upgrading to 13 with a project which includes this package, the following HTML is shown: 

`{"markup":"HTML CONTENT","blocks":{"contentData":[],"settingsData":[]}}`

It seems the new structure for RTE isn't supported yet by the package, we're using a IRteHtmlProcessor to modify the HTML by updating document.DocumentNode.InnerHtml. Is support for Umbraco 13 possible?

**Comment by abjerner** [comment]
Hi @jeroenmink2 

We'll also be needing the package for some of our own projects, so the plan is to update the package to support Umbraco 13. But unfortunately I'm currently caught up in other projects, so I don't have an ETA for when we'll have a release available for Umbraco 13.

**Comment by jeroenmink2** [comment]
Hi @abjerner ,

Is there any update on this?

**Comment by abjerner** [comment]
@jeroenmink2 not really, unfortunately. It was on my todo list at some point, but then our PMs found something they thought I should work on instead. So I don't really know if or when this package will be upgraded 🤷‍♂️
